runtime/interpreter/statictype.go

```diff
@@ -1049,6 +1073,9 @@ func ConvertSemaAccessToStaticAuthorization(
 		if access.Equal(sema.UnauthorizedAccess) {
 			return UnauthorizedAccess
 		}
+		if access.Equal(sema.PrimitiveAccess(ast.AccessNone)) {
```

Comment:
Given the repeated expressions converting AST to sema primitive accesses (e.g. here `sema.PrimitiveAccess(ast.AccessNone)`), it might make sense to define equivalents for the AST accesses (e.g. `ast.AccessNone`) in `sema`, e.g. `const AccessNone =PrimitiveAccess(ast.AccessNone)`, and then replace all occurrences